android/app/src/main/java/org/electroncash/electroncash3/Send.kt

```diff
@@ -199,8 +199,9 @@ class SendDialog : TaskLauncherDialog<Unit>() {
             val inputs = wallet.callAttr("get_spendable_coins", null, daemonModel.config,
                                          Kwarg("isInvoice", pr != null))
             return try {
+                val signSchnorr = daemonModel.walletType == "standard"
```

Comment:
Standard and multisig aren't the only wallet types: see comment in Transactions.kt.